packages/extension-countdown/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@jspsych-contrib/extension-countdown",
+  "private": "true",
```

Comment:
remove private flag so it publishes on npm

Reply:
Made that change. Also I think we should change the `package.json` files in the four templates as they all include `"private": "true"`.

![image](https://github.com/jspsych/jspsych-contrib/assets/74588034/2a513cf1-5030-402f-b53b-1c7339571cc2)

Reply:
My bad. I just realized that we cannot remove that flag in the templates as it would result in them being published on npm. But I still think that deserves to be mentioned in the repo's README.

Reply:
I'm working on a CLI tool in the `add_cli` branch that will take the template and do all the basic replacement work when creating a new plugin or extension. It'll include removing the private flag.